documentation/dev-portal/src/SUMMARY.md

```diff
@@ -68,6 +68,12 @@
 
 # Events
 
+- [cryptotalk](events/cryptotalk/nym-vpn.md)
```

Comment:
why is this all lowercase?

Reply:
This was the info given, I asked to make a doublecheck.